plugins/carbon_black_cloud/icon_carbon_black_cloud/tasks/monitor_alerts/task.py

```diff
@@ -191,7 +199,7 @@ def get_observations(self, job_id: str, state: Dict[str, str]) -> Tuple[list, bo
                 start_observation_time = state.get(LAST_OBSERVATION_TIME)
                 observations, state = self._dedupe_and_get_last_time(observations, state, start_observation_time)
 
-                if observation_json.get("num_found") > PAGE_SIZE:
+                if observation_json.get("num_found") > page_size:
                     self.logger.info("More data is available on the API - setting has_more_pages=True...")
```

Comment:
Can we include what the value of `num_found` is in this logger please? should help us debug how much data is in that queried timeframe